DOC-2481: 7.3 Revision History changes - update plugin page and add release notes (#3407)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/revisionhistory_display_author.adoc

[[revisionhistory_display_author]]
== `revisionhistory_display_author`

> This option configures whether to display the revision's author. When set to `+true+`, the author is displayed in each revision in the Revision History sidebar. If the author's name is not provided, the default value is `Anonymous`.

Reviewer: ```suggestion
This option configures the display of the revision's author. When set to `+true+`, the author's name appears in each revision within the Revision History sidebar. If the author's name is not provided, it defaults to `Anonymous`.
```
